### DISCUSSION TITLE
Pattern in Foocus or --tile in MidJourney

### DISCUSSION THREAD

**Discussion by MarkKoval** [lllyasviel/Fooocus#2070] [Q&A]
Hi everybody!
How I can generate pattern image or something --tile in Foocus?

Using this promt: scribble of moss on rocks pattern --tile

I recieve this, but its not pattern:
![2024-01-27_22-39-33_3766](https://github.com/lllyasviel/Fooocus/assets/45714591/8e3ebfa0-fa72-4c52-87fb-0694a17c1dee)
![2024-01-27_22-40-28_9848](https://github.com/lllyasviel/Fooocus/assets/45714591/e8cafe80-1736-426b-acfe-38aa952ea621)

Else promt: flower pattern

Recieve this and its not a pattern:
![2024-01-27_22-29-24_4291](https://github.com/lllyasviel/Fooocus/assets/45714591/921de2d7-8671-4e9f-b5e3-24cf021bbf0a)
![2024-01-27_22-31-06_8022](https://github.com/lllyasviel/Fooocus/assets/45714591/8ca5b6f5-0ac3-4f96-b342-28dba088b93d)

Maybe someone know how to create pattern?

**Comment by poor7** [comment]
So far, no way. You can create an imitation, but it will not be accurate. I think need a special mechanism is required to check the tail pattern.

**Comment by mashb1t** [comment]
@MarkKoval this is actually a great idea. Currently it's not possible to generate tiled / seamless images in Fooocus, but with the help of https://pypi.org/project/img2texture (source: https://github.com/rtmigo/img2texture#readme) his could be achieved. Feel free to implement this as UOV item!

**Comment by luckynumberke7in** [comment]
I found a quick description of what another person used to create the "tiled" effect (found at the bottom of [this page](https://replicate.com/tommoore515/material_stable_diffusion)):

"_The model uses a circular convolution, so the model sees the image as if all the parallel edges were connected. Like a tube, but in both directions to form a torus:_"
